src-executables/Main-server.hs

```diff
@@ -1,7 +1,7 @@
 {-# LANGUAGE OverloadedStrings #-}
 
 import           Poseidon.GenotypeData       (GenotypeDataSpec (..))
-import           Poseidon.Janno              (PoseidonSample (..))
+import           Poseidon.Janno              (JannoRow (..))
```

Comment:
I love this more explicit name. I should have implemented it like this in the first place!

Reply:
Yeah... I regretted it shortly after I did it :-) But I'm happy now.